### PR TITLE
fix(ugm): create the service

### PR DESCRIPTION
**Pull request by georgettica**

### CODE REVIEW

**Comment by georgettica** [comment]
code is bad, let me fix

**Comment by georgettica** [comment]
last time 🤞

File: pkg/e2e/verify/user-workload-monitoring.go [hunk 1]
```diff
@@ -166,7 +185,7 @@ func newServiceMonitor(name, namespace string) *v1.ServiceMonitor {
 	}
 }
 
-func GenerateService(port int32, serviceName, serviceNamespace string, prometheusName string) (*kv1.Service, error) {
+func GenerateService(port int32, serviceName, serviceNamespace string, prometheusName string, h *helper.H) *kv1.Service {
```
Comment (whereswaldon) [inline pkg/e2e/verify/user-workload-monitoring.go:188]: The helper parameter here is unused. Why add it?

**Review by whereswaldon — CHANGES_REQUESTED** [review]
Everything looks great! I just don't understand one small change.

File: pkg/e2e/verify/user-workload-monitoring.go (hunk 1 above)
Comment (georgettica) [reply pkg/e2e/verify/user-workload-monitoring.go:188]: No reason, and good catch!
I will need a /lgtm one I change this

**Review by whereswaldon — APPROVED** [review]
/lgtm

**Pull request merged**